lmdeploy/turbomind/supported_models.py

```diff
@@ -68,7 +78,12 @@ def is_supported(model_path: str):
     if os.path.exists(triton_model_path):
         support_by_turbomind = True
     else:
-        cfg = AutoConfig.from_pretrained(model_path, trust_remote_code=True)
+        try:
+            cfg = AutoConfig.from_pretrained(model_path,
+                                             trust_remote_code=True)
+        except Exception as e:  # noqa
```

Comment:
what's the excetion?

Reply:
```
transformers/models/auto/configuration_auto.py", line 1130, in from_pretrained
    raise ValueError(
ValueError: The checkpoint you are trying to load has model type `multi_modality` but Transformers does not recognize this architecture. This could be because of an issue with the checkpoint, or because your version of Transformers is out of date.
```
不确定都能handle，可能每个新模型都不一样，transformers 给用户自定义程度挺高的